test/integration/auth/mongodb_oidc_k8s.prose.07.test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from 'chai';
```

Comment:
This prose test does not exist in the spec, but is here just to verify everything works. The spec makes it optional to run all the machine workflow tests across all environments, so in the case of simplicity I just added this one test in alignment with what the Python driver did.